Description: If an external server is used via client.conf and the DNS
 is inconsistent (ex: DNS gives "noname" for many IPs, reverse DNS gives
 one of these IPs for "noname") local PPDs can get polled for print
 queues instead of the PPDs of the external server
Bug: http://www.cups.org/str.php?L2763
Last-Updated: 2014-01-03

--- a/cups/util.c
+++ b/cups/util.c
@@ -1088,6 +1088,8 @@
     http2 = http;
   else if ((http2 = httpConnect2(hostname, port, NULL, AF_UNSPEC,
 				 cupsEncryption(), 1, 30000, NULL)) == NULL)
+  if ((http2 = httpConnect2(http_hostname, http_port, NULL, AF_UNSPEC,
+				 cupsEncryption(), 1, 30000, NULL)) == NULL)
   {
     DEBUG_puts("1cupsGetPPD3: Unable to connect to server");
 
